app/views/galleries/index.rss.builder

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+self.class.send(:include, GalleryDisplayingView)
+
+def gallery_image(gallery)
+  unless gallery.images.count.zero?
+    presenter = presenter_for_gallery_image(gallery.images.first)
+    image_url = presenter.field_value('edmPreview')
+  end
+  image_url ||= @hero_image && @hero_image.file.present? ? @hero_image.file.url : nil
+  image_url
+end
+
+# @todo only insert "views/" into our cache keys when it's needed, i.e. in the context of Mustache view classes
+cache(cache_key(@body_cache_key).sub(%r{\Aviews\/}, ''), skip_digest: true) do
+  xml.instruct!
+  xml.rss(version: '2.0', 'xmlns:atom' => 'http://www.w3.org/2005/Atom', 'xmlns:content' => 'http://purl.org/rss/1.0/modules/content/') do
+    xml.channel do
+      xml.title("Europeana - #{t('global.galleries')}")
+      xml.description(t('site.galleries.description'))
+      xml.link(galleries_url)
+      xml.language(locale.to_s)
+      xml.lastBuildDate(@galleries.maximum(:published_at).rfc2822)
+      xml.tag!('atom:link', rel: 'self', type: 'application/rss+xml', href: galleries_url(format: 'rss'))
+
+      @galleries.each do |gallery|
+        xml.item do
+          xml.title(gallery.title)
+          xml.link(gallery_url(gallery))
+          xml.pubDate(gallery.published_at.rfc2822)
+          xml.guid(gallery_url(gallery))
+          gallery.categorisations.each do |categorisation|
+            xml.category(categorisation.topic_label)
+          end
+          xml.enclosure(url: gallery_image(gallery), length: 0, type: 'image/*')
+          xml.description(gallery.description)
+          xml.content(:encoded) do |content|
+            content.cdata!("<img src=\"#{gallery_image(gallery)}\"/>")
```

Comment:
OK, I understand why you've done this for our internal use case, but I think in future we should adapt our feed consumer to first detect images in dedicated elements like enclosure before looking in content.